kind/deploy-netbox.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
-set -e -u -o pipefail
+set -e -o pipefail
 
 # Deploy NetBox (with its PostgreSQL operator and demo data) into either:
 #  • a local kind cluster (preloading images)
 #  • a virtual cluster using vcluster: https://github.com/loft-sh/vcluster ( used for testing pipeline, loading of images not needed )
 
-NETBOX_HELM_CHART="https://github.com/netbox-community/netbox-chart/releases/download/netbox-5.0.0-beta.169/netbox-5.0.0-beta.169.tgz" # default value
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+
+# Allow override via environment variable, otherwise fallback to default
+NETBOX_HELM_CHART="${NETBOX_HELM_CHART:-https://github.com/netbox-community/netbox-chart/releases/download/netbox-5.0.0-beta.169/netbox-5.0.0-beta.169.tgz}"
```

Comment:
```suggestion
NETBOX_HELM_CHART="${NETBOX_HELM_REPO:-https://github.com/}netbox-community/netbox-chart/releases/download/netbox-5.0.0-beta.169/netbox-5.0.0-beta.169.tgz"
```
like this the version does not need to be updated manually in the environment variable but it's managed by the script.